.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+README.md.expected
```

Comment:
Is this necessary? As I understand it, contributors should run `cargo readme > README.md` before committing and the `README.md.expected` is only used in CI?

Reply:
Not necessary. I added this only so that I do not commit `README.md.expected` by accident while testing. 
If you want I can remove it.